src/views/CreateHospitalView.tsx

```diff
@@ -20,13 +20,18 @@ type PublicProps = RouteComponentProps;
 const defaultHospital = {
   name: '',
   location: {
-    address: '',
+    address: 'asdf',
```

Comment:
Why is the default address `'asdf'` instead of `''`

Reply:
I'll drop that. Not sure where it came from.